src/dnx.clr.managed/DomainManager.cs

```diff
@@ -57,7 +69,50 @@ private int Main(int argc, string[] argv)
         },
         null);
 
-        return RuntimeBootstrapper.Execute(argv);
+        return RuntimeBootstrapper.Execute(argv, _dnxTfm);
+    }
+
+    private Version DetermineDnxVersion(string applicationBase)
```

Comment:
Please measure the perfomance impact on the startup time with this new code before checking it in.